agent/engine/docker_task_engine.go

```diff
@@ -1228,9 +1242,12 @@ func (engine *DockerTaskEngine) provisionContainerResources(task *apitask.Task,
 		}
 	}
 
-	taskIP := result.IPs[0].Address.IP.String()
-	seelog.Infof("Task engine [%s]: associated with ip address '%s'", task.Arn, taskIP)
-	engine.state.AddTaskIPAddress(taskIP, task.Arn)
+	// This is the IP of the task assigned on the bridge for IAM Task roles
```

Comment:
for my understanding, how is windows awsvpc task getting the ip or why it doesn't need the ip?

Reply:
In the feature developed till now, we have not created the bridge for IAM task roles. Therefore, the IP for the task is not available in this feature.
Supporting IAM task roles for Windows awsvpc tasks is the next milestone in this project.